frontend/src/plugins/i18n/index.js

```diff
@@ -109,7 +116,20 @@ Object.defineProperty(i18n, 'browserPreferredLocale', {
         return languageFallback
       }
     }
-    return fallbackLocale
+    if (typeof fallbackLocale === 'string') {
+      return fallbackLocale
+    } else if (Array.isArray(fallbackLocale)) {
+      for (const fallback of fallbackLocale) {
+        return fallback
+      }
+    } else {
+      if ('default' in fallbackLocale) {
```

Comment:
Is this condition ever met? `'default'` is not in our current `fallbackLocale` object as far as I can see, only `['default']` is..?

Reply:
done